.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Notes (delete this section)
+
+* For new APIs, follow http://aka.ms/apireview. This requires filing an issue
+  that outlines the new API first. Before we accept PRs, it muse be reviewed by
+  us and marked as `api-approved`.
```

Comment:
I tried to word it from the perspective of the "project owner", which is basically the people that have push permissions to the repo. I thought it's easier to read than "maintainers".